source/projects/webRadio/de/index.md

```diff
@@ -0,0 +1,136 @@
+
+## RasPi mit dem Radio verbinden
+Die meisten von uns verbinden sich tagtäglich via Bluetooth mit anderen Devices – ganz einfach unter Nutzung des Graphic User Interface (GUI) mit Tastatur und Maus. Das ist in der Regel auch schneller, doch in unserem Workshop stellen wir die Verbindung lediglich mit ein paar Terminal-Command her, denn genau hier liegt die **Magie der Automation**. Speichert man die Commands in einem Shell-Skript und bindet dieses kleine Skript in den Autostart des Raspberry's ein, verbindet sich der Raspberry beim nächsten Hochfahren automagically mit dem Bluetooth-Radio. 
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig (wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss):
+```shell
+$ bluetoothctl
+Agent registered
+
+$ [bluetooth]# power on
+changing power on succeeded
+
+$ [bluetooth]# default-agent
+Default agent request successful
+
+$ [bluetooth]# pairable on
+changing pairable on succeeded
+
+//wichtiger command, falls man sich erneut(!) mit dem gleichen Gerät verbinden möchte
+$ [bluetooth]# remove 12:34:56:78:9A:BC
+changing pairable on succeeded
+
+// Umgebung nach Bluetooth-Devices scannen; MAC-Adresse des Radios sollte erscheinen
+$ [bluetooth]# scan on
+Discovery started
+[CHG] Device 12:34:56:78:9A:BC 12-34-56-78-9A-BC
+
+$ [bluetooth]# scan off
+Discovery stopped
+
+//mit dem Radio pairen (!= verbinden)
+$ [bluetooth]# pair 12:34:56:78:9A:BC
+Attempting to pair with 12:34:56:78:9A:BC
+[CHG] Device 12:34:56:78:9A:BC Connected: yes
+[CHG] Device 12:34:56:78:9A:BC ServicesResolved: yes
+[CHG] Device 12:34:56:78:9A:BC Paired: yes
+Pairing successful
+
+//überprüfen ob das Pairing wirklich geklappt hat; Radio sollte gelistet werden
+$ [bluetooth]# devices
+Device 12:34:56:78:9A:BC MeinRadioName
+
+//automatisch mit dem Gerät (Radio) verbinden, wenn es eingeschaltet wird
+$ [bluetooth]# trust 12:34:56:78:9A:BC
+[CHG] Device 12:34:56:78:9A:BC Trusted: yes
+Changing 12:34:56:78:9A:BC trust succeeded
+
+//Finally: mit dem Radio verbinden
+$ [bluetooth]# connect 12:34:56:78:9A:BC
+```
+
+## Web Radio per Autostart abspielen
+
+Damit der Aha-Effekt größer und das Ergebnis hörbar ist, haben wir das Shell-Skript um einen weiteren Command ergänzt. Mit dem Command
+```shell
+$ vlc meinePlaylist.m3u
+```
+öffnet der bereits vorinstalliert VLC-Mediaplayer die eigens erstellte Playlist "meinePlaylist". Alle darin befindlichen Dateien wie mp3-Dateien oder weitere Playlists, beispielsweise die Playlist eures Lieblings-Web-Radios, werden mit dem Command nacheinander abgespielt. In die Playlist muss demnach lediglich eine URL zu entsprechenden Web-Radiosender hinterlegt werden.
+
+**Last but not least: das Shell-Skript in den Autostart einbinden.** Um ein Programm in den Autostart des Raspberry's einzubinden gibt es verschiedene Möglichkeiten. In unserem Tutorial haben wir eine **.dektop-Datei** erstellt, um unser Skript mit jedem Hochfahren des RasPi's auszuführen. 
```

Comment:
~~**.dektop-Datei**~~ **.desktop-Datei**

Reply:
It is still `dektop` not `desktop`

Reply:
finally ;)